experimenter/experimenter/experiments/models.py

```diff
@@ -790,6 +790,38 @@ def review_url(self):
                 review_path="simple-review",
             )
 
+    @property
+    def audience_url(self):
+        filters = [
+            ("application", self.application),
+        ]
+        if self.channel:
+            filters.append(("channel", self.channel))
+        if self.firefox_min_version:
```

Comment:
do we want to check max version too?

Reply:
I don't think so, since we just want to know the most recent matching number, and either some version is live and above the minimum so clients are still enrolled and that's the number we care about, or they've passed the max and gone to zero in which case we don't care about those numbers.  If we decide we do want to filter on it we can add it later, but I think this is all we need for now.